ext/drwrap/drwrap.c

```diff
@@ -974,6 +974,13 @@ drwrap_exit(void)
     if (count != 0)
         return;
 
+    drmgr_unregister_bb_app2app_event(drwrap_event_bb_app2app);
```

Comment:
I don't understand how these affect stability: while it is good to clean up, I thought they did not matter to the bottom line when exiting as drmgr and DR throw out the registration lists at exit time anyway.  Please update #3049 with details on why failing to unregister at exit time causes a problem.

Reply:
I have updated #3049 with more details.